frontend/src/bundles/common/components/user-info-card/user-info-card.tsx

```diff
@@ -48,11 +49,7 @@ const UserInfoCard: React.FC<Properties> = ({
                         {name}
                     </p>
                 </div>
-                <img
-                    src={image}
-                    alt={name}
-                    className="h-44 w-44 overflow-hidden rounded-full object-cover"
-                />
+                <Avatar size="xl" email={image ? '' : name} avatarUrl={image} />
```

Comment:
```suggestion
                <Avatar size="xl" email={name} avatarUrl={image} />
```
You can pass just name because Avatar have this condition, if there is no image, name will be used for displaying.